ydb/docs/ru/core/devops/deployment-options/manual/toc_p.yaml

```
@@ -14,3 +14,7 @@ items:
   include:
     mode: link
     path: before-v25.1/toc_p.yaml
+- name: Миграции
```

index.md появляется, но в навигацию как-будто не подключен
```suggestion
- name: Миграции
   href: migration/index.md
```